### ISSUE TITLE
Consider trimming spaces in parsed html

### ISSUE THREAD

**Issue by f0rr0** [martinheidegger/excerpt-html#2]
If the html contains inline images (or some other html entity with non text content) with spaces in between, the parsed text contains those spaces. 

``` html
<p><img></img> <img></img> <img></img> text</p>
```

would produce `<space><space><space>text` after parse.

These spaces count as characters in the `pruneLength`.

A simple regex to trim this could be

``` js
parsed = parsed.replace(/^\s+|\s+$|\s+(?=\s)/g, "")
```

**Comment by martinheidegger** [comment]
Hello @f0rr0 

Great suggestion! It would be nice if you could contribute a PR? Would you be open to help out?

**Comment by f0rr0** [comment]
@martinheidegger done!

**Comment by martinheidegger** [comment]
@f0rr0 Thanks! I added a test in https://github.com/martinheidegger/excerpt-html/commit/5e1d0a35987a9c42bf7f4f98045a779d9daf48ff and published it as 1.2.0 Awesome work!
